go/test/endtoend/vreplication/vdiff2_test.go

```diff
@@ -267,6 +318,7 @@ func testCLIFlagHandling(t *testing.T, targetKs, workflowName string, cell *Cell
 			AutoRetry:             true,
 			UpdateTableStats:      true,
 			TimeoutSeconds:        60,
+			MaxDiffSeconds:        333,
```

Comment:
333 seems like an oddly-specific duration to me ... is there a reason you went with this?

Reply:
Only in that I needed to pick something specific and this is the random number I typed. 🙂